### ISSUE TITLE
[Usage]: how can i get all logits of token?

### ISSUE THREAD

**Issue by joyyyhuang** [vllm-project/vllm#10149]
### Your current environment

```text
The output of `python collect_env.py`
```

### How would you like to use vllm

I want to do a discriminant task and directly use the logits score of the last layer, such as comparing the logits score of the two tokens yes or no, but I designed Qwen to only output one character through prompt design. How can I get the logits score of the above two characters.

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by Hukongtao** [comment]
> 我修改了下model_executor/layers/sampler.py，在Sample the next tokens之后，重置logprobs = logits。不够优雅，但是推理结果可以返回logits了。可以参考，不知道会不会对生成结果有影响。 #10688

能否会整个向量的logits吗？而不是logits里面的最大值。你这种修改返回的还是logits里面的最大值吧

**Comment by wbchief** [comment]
> 我修改了下model_executor/layers/sampler.py，在Sample the next tokens之后，重置logprobs = logits。不够优雅，但是推理结果可以返回logits了。可以参考，不知道会不会对生成结果有影响。 #10688

你好有验证对结果影响吗

**Comment by DarkLight1337** [comment]
Please refer to the more detailed discussion in https://github.com/vllm-project/vllm/issues/11397.
